### DISCUSSION TITLE
Customtkinter Frames

### DISCUSSION THREAD

**Discussion by peer-pri** [TomSchimansky/CustomTkinter#1447] [General]
If  you take a look at the examles and than image_example.py. There is my problem. I deleted the buttons. Instead i want text there. Labels. How to make different labels for home, frame1 and frame 2? Please help fast

**Comment by VasigaranAndAngel** [comment]
https://github.com/TomSchimansky/CustomTkinter/blob/09e584634c867f2b6074fcbfe41cba80d2b78c66/examples/image_example.py#L57-L71

If you pack any widgets inside `self.home` widget, it will be inside the home frame.
same as that,

https://github.com/TomSchimansky/CustomTkinter/blob/09e584634c867f2b6074fcbfe41cba80d2b78c66/examples/image_example.py#L73-L77

if you pack any widgets inside `self.second_frame` or `self.third_frame` it will be inside the frame1 and frame2 respectively.

This will help you to understand - https://youtu.be/_auZ8TTkojQ
